source/patterns/@aws-solutions-constructs/aws-events-rule-kinesisfirehose-s3/test/integ.events-rule-kinesisfirehose-s3-existing-eventbus.ts

```diff
@@ -26,9 +28,23 @@ const props: EventsRuleToKinesisFirehoseToS3Props = {
       source: ['solutionsconstructs']
     }
   },
-  existingEventBusInterface: existingEventBus
+  existingEventBusInterface: existingEventBus,
+  bucketProps: {
+    removalPolicy: RemovalPolicy.DESTROY
+  },
+  logGroupProps: {
+    removalPolicy: RemovalPolicy.DESTROY
+  },
+  logS3AccessLogs: false
```

Comment:
Why are we turning off Access logs entirely for our integration tests?

Reply:
To fully clean up our buckets you mentioned to either turn off logging or add removalPolicy for our tests. Turning off logging would prevent creating additional integ.noLoggingBucket.ts file since it would already be tested in the other integ tests. Do you think it would be better to have integ.noLoggingBucket.ts and leave logging on or just integrate it with current tests?